server/addPerson.js

```diff
@@ -15,8 +16,13 @@ module.exports = wrapper(async (req, client) => {
   const user = await getPersonByEmail(client, userEmail);
   if (user) return {status: 401, data: 'User already exists!'};
 
+  //This method of hashing is sync and it's safer to use async, though for this
+  //tiny project I don't think it's ever going to be a problem.
+
+  const hash = bcrypt.hashSync(userPassword, 10);
```

Comment:
Any reason to not do async? I think it would be `const hash = await bcrypt.hash(userPassword, 10);`